lib/minify.js

```diff
@@ -10,6 +10,10 @@ exports.simple_glob = function(files) {
     return files;
 };
 
+var defaultBase64Decoder = exports.base64Decoder = function(input) {
```

Comment:
Can the function be implemented as follows so the default works in both `Node.js` and browsers?
```js
exports.base64Decode = function base64Decode(data) {
    return typeof atob !== 'undefined' ? atob(data) : Buffer(data, 'base64').toString();
}
```